lib/course_planner/tasks.ex

```diff
@@ -47,22 +47,32 @@ defmodule CoursePlanner.Tasks do
     end
   end
 
-  def get_user_tasks(id) do
-    Repo.all(from t in Task, where: t.user_id == ^id, preload: [:user])
+  def get_for_user(id, now) do
+    Repo.all(from t in Task, where: t.user_id == ^id and t.finish_time > ^now, preload: [:user])
```

Comment:
I understand hiding the past unassigned tasks, but why would we hide the assigned ones? what if I want to check which tasks I did last week for example?